aem-authoring-toolkit-plugin/src/test/com/exadel/aem/toolkit/test/widget/Accordion.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.exadel.aem.toolkit.test.widget;
+
+import com.exadel.aem.toolkit.api.annotations.container.AccordionPanel;
+import com.exadel.aem.toolkit.api.annotations.container.PlaceOn;
+import com.exadel.aem.toolkit.api.annotations.main.Dialog;
+import com.exadel.aem.toolkit.api.annotations.widgets.AccordionWidget;
+import com.exadel.aem.toolkit.api.annotations.widgets.DialogField;
+import com.exadel.aem.toolkit.api.annotations.widgets.TextField;
+import com.exadel.aem.toolkit.api.annotations.widgets.button.Button;
+import com.exadel.aem.toolkit.api.annotations.widgets.button.ButtonType;
+import com.exadel.aem.toolkit.api.annotations.widgets.common.ElementSizeConstants;
+
+import static com.exadel.aem.toolkit.core.util.TestConstants.DEFAULT_COMPONENT_NAME;
+
+@Dialog(
+        name = DEFAULT_COMPONENT_NAME,
+        title = "Accordion Test Dialog",
+        panels = {
+                @AccordionPanel(title = "Basic", disabled = true),
+                @AccordionPanel(title = "Basic2")
+        }
+)
+@SuppressWarnings("unused")
+public class Accordion {
+
+    @Button(
```

Comment:
(Tactical). Do not add so many detail to an annotation which is not being tested in this particular test case. This will only make the XML sample bigger and distract attention from what is actually being tested. It's better to use "placeholder" widgets that have as few fields as possible. Kind of TextField or even Checkbox

Reply:
Fixed